src/Message/Message.module.scss

```diff
@@ -0,0 +1,61 @@
+@import '../variables.scss';
+@import '../mixins.scss';
+
+.wrapper {
+  @include text-normal;
+
+  height: 40px;
+  border: 1px solid;
+  box-sizing: border-box;
+  border-radius: $border-radius;
+  display: flex;
+  align-items: center;
+  padding: 12px;
+
+  svg {
```

Comment:
@DmitriVanGuard so do you think the very first implementation consisting classes like `icon, icon-warn, icon-alert...` was better and I should revert back to that?

Reply:
@devanshu-santiment please, reread [this](https://github.com/santiment/san-ui/pull/35/files/2fe3ea2f088e45e4a67e056e99185346da29d7fd#r260183465) comment closely. Never mentioned that independent `.icon` selector should be removed.

Just rewrite it like this:
instead of 
```scss
.wrapper {
// ...
   svg {
      // icon styles
   }
}
```
write this:
```scss
.icon {
      // icon styles
}
```